lang/en.json

```
@@ -1410,6 +1410,7 @@
 "DND5E.SpellPrepAlways": "Always Prepared",
 "DND5E.SpellPreparation": "Spell Preparation",
 "DND5E.SpellPreparationMode": "Spell Preparation Mode",
+"DND5E.BoundSpellOrigin": "Bound spellcasting origin",
```

I think it should be something a bit more specific referencing the class:

```suggestion
"DND5E.SpellSourceClass": "Source Class",
```

(also it should be shifted below "DND5E.SpellSlotsConfig" to keep some semblance of alphabetical order)